### PR TITLE
chore: prepare for v10.x

### PR DESCRIPTION
**Pull request by owenvoke**
Depends on https://github.com/laravel-zero/framework/pull/471, https://github.com/laravel-zero/foundation/pull/88, and https://github.com/laravel-zero/docs/pull/29

### CODE REVIEW

File: app/Commands/InspireCommand.php [hunk 1]
```diff
@@ -41,11 +41,8 @@ public function handle()
 
     /**
      * Define the command's schedule.
-     *
-     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
-     * @return void
      */
-    public function schedule(Schedule $schedule)
+    public function schedule(Schedule $schedule): void
```
Comment (nunomaduro) [inline app/Commands/InspireCommand.php:45]: The method handle should be typed too - to `void`.

**Review by nunomaduro — CHANGES_REQUESTED** [review]

File: app/Commands/InspireCommand.php (hunk 1 above)
Comment (owenvoke) [reply app/Commands/InspireCommand.php:45]: Hmm. I thought it could be `void` or `int`? (I have various commands where I return the status code)

Comment (nunomaduro) [reply app/Commands/InspireCommand.php:45]: `void` for the inspire command.

Comment (nunomaduro) [reply app/Commands/InspireCommand.php:45]: In other words, `void` by default. But people can change it to `int` if they want.

Don't forget to also type hint the documentation.

Comment (owenvoke) [reply app/Commands/InspireCommand.php:45]: 🤦🏻 Thought this was in the framework, will make the changes tomorrow (later today). 👍🏻

**Comment by funder7** [comment]
Hi, is this merge already scheduled, or .... ? cheers

**Comment by owenvoke** [comment]
Hi @funder7, Laravel Zero usually releases about a week after Laravel core does. Hoping to run through the last few checks today and get it out soon though. 👍🏻

**Comment by funder7** [comment]
> Hi @funder7, Laravel Zero usually releases about a week after Laravel core does. Hoping to run through the last few checks today and get it out soon though. 👍🏻

Okay! Meanwhile I'm using this config (development env. only):
image

Is this 10.x already, or it's a migration work in progress?

**Pull request merged**